### PR TITLE
Fix linters in CI

### PR DESCRIPTION
**Pull request by paulfantom**
- remove JS linting as we have codacy and LGTM
- unify mountpoints to always use `/code` directory
- add `python.d.plugin.in` to be linted with flake8

Relates to #4391

### CODE REVIEW

**Comment by paulfantom** [comment]
Approve/Merge?

**Comment by ktsaou** [comment]
hm... what is missing?

File: .travis.yml [hunk 1]
```diff
@@ -21,16 +21,13 @@ jobs:
     - stage: "lint"
       name: "python"
       install: skip
-      script: docker run -ti --rm -v $(pwd):/apps alpine/flake8:3.5.0 --max-line-length=120 --exclude=third_party,urllib3,pyyaml3,pyyaml2 python.d/ plugins.d/python.d.plugin
-    - name: "javascript"
-      install: skip
-      script: docker run -it --rm -v $(pwd)/web:/code eeacms/jslint --color /code/web/*.js /code/plugins.d/node.d.plugin/*.js /code/node.d/*.js /code/node.d/node_modules/netdata.js
+      script: docker run -ti --rm -v $(pwd):/code alpine/flake8:3.5.0 --max-line-length=120 --exclude=third_party,urllib3,pyyaml3,pyyaml2 /code/collectors/python.d.plugin/python.d.plugin.in /code/
```
[the remark this answers is not in the document]
Comment (paulfantom) [reply .travis.yml]: top level directory for netdata

**Comment by paulfantom** [comment]
Review from @Ferroin and/or @l2isbad.

**Comment by ktsaou** [comment]
ok

**Pull request merged**